### PR TITLE
Inline mode

### PR DESCRIPTION
**Pull request by MarcoBuster**
Finally, the complete support for inline mode, with a very nice API:
```python
@bot.inline
def inline_processor(inline):
    yield inline.article(
        "Hello World",
        content=botogram.InlineInputMessage("My first inline response with botogram")
    )
```

We're very sorry for the delay, we had personal issues and other work. Maybe during this period in quarantine we can work more (stay safe and stay the fuck home, btw)!

### CODE REVIEW

**Comment by MarcoBuster** [comment]
Only [ChosenInlineResult](https://core.telegram.org/bots/api#choseninlineresult) update is missing, @matteob99 is working on this now.

File: docs/api/telegram.rst [hunk 1]
```diff
@@ -3144,7 +3192,9 @@ about its business.
       vast majority of API objects.
 
       *This attribute can be None if it's not provided by Telegram.*
+      .. versionchanged:: 0.7
 
+         Can be None.
```
Comment (matteob99) [inline docs/api/telegram.rst]: delete this line

File: docs/inline.rst [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+  def inline_processor(inline):
+      btns = botogram.buttons()
+      btns[0].url("botogram docs", "https://botogram.dev/")
+      btns[0].callback("Click on this button", "button")
+      yield inline.article("Hello World", content=botogram.InlineInputMessage("Hello World message"), attach=btns)
+
+
+  @bot.callback("button")
+  def button_callback(message):
+  if message.is_inline:
```
Comment (matteob99) [inline docs/inline.rst]: wrong indentation

File: docs/inline.rst [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+      yield inline.article(f"Your ID: {sender.id}", botogram.InlineInputMessage("This message is cached only for you"))
+      inline.private = False
+      yield inline.article("Hello World", botogram.InlineInputMessage("This message is cached for all users"))
+
+
+Working with buttons
+-----------------
+
+In the following example you can see how buttons works with inline mode, plus some differences compared to non-inline buttons.
+
```
Comment (matteob99) [inline docs/inline.rst]: missing "For more details see"

File: docs/inline.rst [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+Inline feedbacks
+----------------
+
+If you have many users, maybe you want to acquire statistics or do something about which elements are chosen by the user
+in the inline results page.
+You can specify how many (in percentage) inline feedbacks do you want to get via BotFather with the ``/setinlinefeedback`` command;
+you'll then receive an update when the user *clicks* on a inline result.
+
+For more details see :py:class:`~botogram.InlineFeedback`.
```
Comment (matteob99) [inline docs/inline.rst]: py class InlineFeedback is not linked

File: docs/inline.rst (hunk 3 above)
Comment (MarcoBuster) [reply docs/inline.rst]: See what? What do you mean?

**Pull request merged**